app/components/svg-image.jsx

```diff
@@ -1,4 +1,21 @@
-import React, {PropTypes} from 'react';
+import React from 'react';
+
+const FILTERS = {
+    invert: "url('#svg-invert-filter')",
+}
```

Comment:
Missing semicolon at the end of this constant. Also, line four is indented too much.